### DISCUSSION TITLE
how to change things like distance units

### DISCUSSION THREAD

**Discussion by Gazzxy** [YetiForceCompany/YetiForceCRM#14949] [Feature improvement]
is there a way to change distance units ? currently things like kilometres are used when we need miles

**Comment by mariuszkrzaczkowski** [comment]
attach a screenshot where you want to change it

**Comment by Gazzxy** [comment]
Here... but to be fair.. anywhere that refers to distance should ideally be miles
![image](https://user-images.githubusercontent.com/83064841/125291831-bb3da900-e319-11eb-99d9-cd6a209873c1.png)

**Comment by diderich** [comment]
You could change "kilometer" in the language files/translations to "miles" as a quick fix.

**Reply by Gazzxy** [reply]
yeh that could work.. see what happens when people start using it
